vllm/model_executor/models/llama.py

```diff
@@ -257,11 +257,12 @@ def __init__(
                       (lora_config.max_loras or 1)) if lora_config else 0
         self.vocab_size = config.vocab_size + lora_vocab
         self.org_vocab_size = config.vocab_size
-        self.embed_tokens = VocabParallelEmbedding(
-            self.vocab_size,
-            config.hidden_size,
-            org_num_embeddings=config.vocab_size,
-        )
```

Comment:
How about adding one line?

`self.embed_tokens = PPMissingLayer()`

The `PPMissingLayer` exists in `vllm.model_executor.models.utils`